lib/beaker/dsl/install_utils/foss_utils.rb

```diff
@@ -367,6 +367,12 @@ def install_puppet_agent_on(hosts, opts = {})
             end
 
             if package_name
+              # Some SLES images already have rubygem-puppet installed and this
```

Comment:
@tvpartytonight I've been using the images directly from SuSE and OpenSuSE. If you take that out...you're gonna have a bad time....sometimes....